src/SDK/Logs/SimplePsrFileLogger.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Logs;
+
+use Psr\Log\InvalidArgumentException;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LoggerTrait;
+use Psr\Log\LogLevel;
+use ReflectionClass;
+use Throwable;
+
+class SimplePsrFileLogger implements LoggerInterface
```

Comment:
in a container world (docker, k8s), logging to stdout and stderr is the norm. Since you can file_put_contents to php://stdout, this could become `SimplePsrStreamLogger`, and default to stdout?

Reply:
To be honest I'd rather rename this logger to `DoNotUseInProductionSeriouslyLogger`. It is just meant to be used for demo/example purposes or at max local development. It has some serious design flaws. Logging to stdout/err is easy and comfortable, but it hides a lot of complexity, as well as K8S does, and at the end of the day even "high availabilty" only means the system doesn't crash, just because you look at it. However this little logger implementation has the potential to create an indefinit amount of backpressure, as it sends everything to the configured destination, The log level is just a "tag" for it.
So I'd rather not recommend anyone to use it in any serious application, but use a battle tested and well maintained solution like Monolog, which comes with a lot of configuration options, diferrent [handlers, filters](https://github.com/Seldaek/monolog/blob/main/doc/02-handlers-formatters-processors.md), and so on.

Also this logger will not make it easier to implement enrichment of logs with tracing data, etc. in the future. It's easier to provide contrib plugins for popular thirds party loggers, than to reinvent the wheel in that case, especially because people will use those libraries anyway.

But these are just my thoughts, feel free to change the logger once the PR is merged.